pkg/tcpip/header/icmpv4.go

```diff
@@ -49,6 +52,9 @@ const (
 	// icmpv4SequenceOffset is the offset of the sequence field
 	// in a ICMPv4EchoRequest/Reply message.
 	icmpv4SequenceOffset = 6
+	
+	// Milli Seconds in a day calculation for Timestamp reply messages
```

Comment:
The comment should reflect the actual variable name. E.g.

MsInDay is ...

Also, please use full sentences for all comments and end in a ‘.’.

(I have feedback about the actual naming below.)